tests/configs/bad/typos.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	sentry: "go",
```

Comment:
Eventually we'll add some kind of fancy "you may have meant" feature to get this right (should be "entry")